frontend/@mf-types/@mf/modelRegistry/compiled-types/src/app/api/updateTimestamps.d.ts

```
@@ -0,0 +1,6 @@
+import { ModelRegistryAPIs, ModelVersion, RegisteredModel } from '~/app/types';
+type MinimalModelRegistryAPI = Pick<ModelRegistryAPIs, 'patchRegisteredModel'>;
+export declare const bumpModelVersionTimestamp: (api: ModelRegistryAPIs, modelVersion: ModelVersion) => Promise<void>;
+export declare const bumpRegisteredModelTimestamp: (api: MinimalModelRegistryAPI, registeredModel: RegisteredModel) => Promise<void>;
+export declare const bumpBothTimestamps: (api: ModelRegistryAPIs, registeredModel: RegisteredModel, modelVersion: ModelVersion) => Promise<void>;
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)